src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphAuthDAOSpec.scala

```diff
@@ -329,7 +338,7 @@ class GraphAuthDAOSpec extends FlatSpec with Matchers with OrientDbTestFixture {
           WorkspaceAccessLevels.groupAccessLevelsAscending map { level =>
             val levelFromWs = authDAO.getVertices(mapVertex, Direction.OUT, EdgeSchema.Ref, level.toString).head
 
-            val levelGroup = RawlsGroup(UserAuth.toWorkspaceAccessGroupName(testData.workspace.toWorkspaceName, level), Set.empty, Set.empty)
+            val levelGroup = RawlsGroup(testData.workspace.toWorkspaceName, level, Set.empty[RawlsUserRef], Set.empty[RawlsGroupRef])
```

Comment:
Parameterizing `Set.empty` makes me sad, but I was not clever enough to avoid doing so with the `apply`